app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def index
+    @results = FoodFacade.foods_by_ingredient(params[:q])
```

Comment:
Love that this is one line, but I might call the instance variable `@search` to match the name of your object.